src/jvm/test/org/jetbrains/kotlinx/lincheck/test/util/TestUtils.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Lincheck
+ *
+ * Copyright (C) 2019 - 2023 JetBrains s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>
+ */
+
+package org.jetbrains.kotlinx.lincheck.test.util
+
+import org.jetbrains.kotlinx.lincheck.*
+import org.jetbrains.kotlinx.lincheck.strategy.managed.modelchecking.*
+import org.junit.Assert.*
+
+internal fun Any.runModelCheckingTestAndCheckOutput(
```

Comment:
Please document the function (e.g., it is unclear what "Any" is).

Reply:
Done

Reply:
In the model checking mode? Please specify explicitly what this "Any" is (the test instance?).

Reply:
Please also check the KDoc syntax: https://kotlinlang.org/docs/kotlin-doc.html

Reply:
Done